src/index.ts

```diff
@@ -5,13 +5,17 @@ import { GameView } from './examples/chess/GameView'
 import { createOrderBuilder } from './examples/order/builder'
 import { OrderView } from './examples/order/OrderView'
 import { Welcome, createNullBuilder } from './examples/welcome'
+import { CounterView } from './examples/counter/CounterView'
+import { createCounterBuilder } from './examples/counter/builder'
```

Comment:
Please add `examples/counter/index.ts`.

I should have done this for `examples/order` as well but I was in a bit of a rush.